logstash-core/src/test/java/org/logstash/common/io/DeadLetterQueueReaderTest.java

```diff
@@ -134,9 +134,63 @@ public void testSeek() throws Exception {
         assertThat(entry.getReason(), equalTo("543"));
     }
 
+
     @Test
-    public void testInvalidDirectory()  throws Exception {
-        DeadLetterQueueReader reader = new DeadLetterQueueReader(dir);
-        assertThat(reader.pollEntry(100), is(nullValue()));
+    public void testWriteStopSmallWriteSeekByTimestamp() throws Exception {
+        DeadLetterQueueWriter writeManager = new DeadLetterQueueWriter(dir, 10000000, 10000000);
+        Event event = new Event(Collections.emptyMap());
+        Timestamp target = null;
```

Comment:
@robbavey can't we just move this below the `currentEpoch` line and do ?:

```java
final target = new Timestamp(currentEpoch + 100)
```

... instead of the redundant reassignment of `target` in the loop below.

(also we could extract the `100` to a variable then and make things a little easier to follow)